sdk/eventhub/eventhubs-checkpointstore-blob/src/blobCheckpointStore.ts

```diff
@@ -113,14 +114,17 @@ export class BlobCheckpointStore implements CheckpointStore {
           // etag failures (precondition not met) aren't fatal errors. They happen
           // as multiple consumers attempt to claim the same partition (first one wins)
           // and losers get this error.
-          log.blobCheckpointStore(`[${ownership.ownerId}] Did not claim partition ${ownership.partitionId}. Another processor has already claimed it.`);
+          logger.verbose(
+            `[${ownership.ownerId}] Did not claim partition ${ownership.partitionId}. Another processor has already claimed it.`
+          );
           continue;
         }
 
-        log.error(
+        logger.warning(
           `Error occurred while claiming ownership for partition: ${ownership.partitionId}`,
-          err
+          err.message
```

Comment:
Will they need the name of the error? I think _not_ but I just want to check.

Reply:
I could add it, but since it's most likely a `RestError` I just went with the message. You will see the name in the stack trace if you enable verbose logging.